### PR TITLE
Addressed Issue 1.

### PR DESCRIPTION
**Pull request by willj4945**
80% Completion on Issue 3
Have not started issue 2

### CODE REVIEW

**Comment by willj4945** [comment]
Issue 1 is resolved.
Issue 2 has not been started
Issue 3 is 80% resolved

**Comment by JBraunsmaJr** [comment]
The usual approach is 1 of two ways.

1. Pick an issue to work on (just one) and complete them in a separate branch. This is called a `feature set` branch. Once completed, you create a pull request. Then it gets reviewed by team members and when all sign off.

2. Similar to what you did, if there are a lot of relatively-small changes... super small it's acceptable to complete them together in 1 feature-set branch / PR. However, it should be the entirety of whatever you were going after. If something is 80% done it should not get merged until completion.

File: css/styles.scss [hunk 1]
```diff
@@ -162,7 +162,20 @@ section {
     position: relative;
 
     }
+  .profile-image{
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    margin-left: 4rem;
+    padding-top: 2rem;
```
Comment (JBraunsmaJr) [inline css/styles.scss:170]: So, you're not necessarily wrong here. This does move the content away from the top/left parts of the screen. However, it's not reusable and other content in the DOM would still touch the edges of the window.

Could look something like this where you apply a class on a parent (see the html part for more info)
```css
.container {
   padding: 16px;
}
```

File: index.html [hunk 2]
```diff
@@ -32,26 +32,26 @@
     <div class="header-content">
```
Comment (JBraunsmaJr) [inline index.html:32]: I am trying so hard to highlight line 30  but it won't let me

```html
<body class="main-content">
```

This is one place you could add the container / padding which would impact everything you see on screen.

However, it's worth noting it impacts ALL the things. Some people may implement a navigation bar in their `body` which would mean the padding would also get that padding... giving a whacky look.

In your case your side nav could be impacted but you could adjust it as needed. Or just use the css class on a lower level element to get the desired effect. Just want to make sure your text is NOT touching the sides of the window.

**Pull request merged**